src/components/NcWMSAutosetColorscaleControl/LeafletNcWMSAutosetColorscaleControl.js

```diff
@@ -0,0 +1,76 @@
+import L from 'leaflet';
```

Comment:
Wow, we need a shorter name for this module.

Reply:
I? :)

Reply:
I meant a shorter name for ``LeafletNcWMSAutosetColorscaleControl``. I mean... it *is* descriptive. And I'm terrible at naming things, so I have not alternate suggestions.